src/components/visualizations/chart/highcharts/plugins/adjustTickAmount.ts

```diff
@@ -125,32 +125,45 @@ export function adjustTicks(axis: IHighchartsAxisExtend): void {
 }
 
 /**
- * Align zero of secondary axis to primary axis
+ * Get axis score that increase 1 for data having positive and negative values
  * @param axis
```

Comment:
Please add more information to this `@param` and add `@return`
Or you can remove it completely 
This should apply for other function comments

Reply:
done